### PR TITLE
TIS-740/select quay multiple times

### PR DESCRIPTION
**Pull request by solita-sabinaf**

### CODE REVIEW

File: src/ext/JourneyPatternStopPointMap/JourneyPatternStopPointMap.tsx [hunk 1]
```diff
@@ -10,10 +10,10 @@ import { getStopPlaces } from '../../actions/stopPlaces';
 import { useDispatch } from 'react-redux';
 import { StopPointLocation } from '../../reducers/stopPlaces';
 import { StopPlace } from '../../api';
-import QuaysWrapper from './QuaysWrapper';
+import QuaysWrapper from './Quay/QuaysWrapper';
 
 interface MapState {
-  quayStopPointIndexRecord: Record<string, number>;
+  quayStopPointIndexesRecord: Record<string, number[]>;
```
Comment (solita-sabinaf) [inline src/ext/JourneyPatternStopPointMap/JourneyPatternStopPointMap.tsx:16]: Rename

File: src/ext/JourneyPatternStopPointMap/Quay/QuayMarker.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+          hasNonSelectedQuays={hasNonSelectedQuays}
+          hideNonSelectedQuaysState={hideNonSelectedQuaysState}
+          hideNonSelectedQuaysCallback={hideNonSelectedQuaysCallback}
+          showQuaysCallback={showQuaysCallback}
+          addStopPointCallback={addStopPointCallback}
+        />
+      </Popup>
+
+      {stopPointIndexes?.length > 0 && (
+        <QuayIndexTooltip
```
Comment (solita-sabinaf) [inline src/ext/JourneyPatternStopPointMap/Quay/QuayMarker.tsx:81]: Rename

File: src/ext/JourneyPatternStopPointMap/Quay/QuayPositionChips.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            <TagChip
+              className={'stop-point-index-chip'}
+              size={'small'}
+              onClose={() => {
+                setDeleteDialogOpen(true);
+              }}
+            >
+              {index + 1}
+            </TagChip>
+            <ConfirmDialog
```
Comment (solita-sabinaf) [inline src/ext/JourneyPatternStopPointMap/Quay/QuayPositionChips.tsx:40]: Rethink this

File: src/i18n/translations/en.ts [hunk 4]
```diff
@@ -637,8 +637,9 @@ export const messages: MessagesKey = {
   oneQuay: '1 quay',
   numberOfQuays: '{count} quays',
   addToJourneyPattern: 'Add',
-  removeFromRoute: 'Remove from route',
   quayTitle: 'Quay {name}',
   showNonSelectedQuays: 'Show non-selected quays',
   hideNonSelectedQuays: 'Hide non-selected quays',
+  quayPosition: 'Position',
```
Comment (solita-sabinaf) [inline src/i18n/translations/en.ts:643]: Rename

File: src/ext/JourneyPatternStopPointMap/JourneyPatternStopPointMap.tsx (hunk 1 above)
Comment (solita-sabinaf) [reply src/ext/JourneyPatternStopPointMap/JourneyPatternStopPointMap.tsx]: I btw checked, both indices and indexes are valid 😄 Thought to keep as indexes so it's faster to recognise

File: src/i18n/translations/nb.ts [hunk 5]
```diff
@@ -635,8 +635,8 @@ export const messages = {
   oneQuay: '1 quay',
   numberOfQuays: '{count} quayer',
   addToJourneyPattern: 'Legg til',
-  removeFromRoute: 'Fjern',
   quayTitle: 'Quay {name}',
   showNonSelectedQuays: 'Vis ikke valgte quayer',
   hideNonSelectedQuays: 'Skjul ikke valgte quayer',
+  quayOrder: 'Rekkefølgen',
```
Comment (testower) [inline src/i18n/translations/nb.ts:641]:
```suggestion
  quayOrder: 'Rekkefølge',
```

**Review by testower — CHANGES_REQUESTED** [review]

**Pull request merged**